interfaces/console/Dockerfile

```diff
@@ -1,62 +1,27 @@
-# FROM nginx
-# COPY  build /usr/share/nginx/html
-# RUN rm /etc/nginx/conf.d/default.conf
-# COPY nginx/nginx.conf /etc/nginx/conf.d
-# EXPOSE 8080
-# CMD ["nginx", "-g", "daemon off;"]
+FROM node:18-alpine AS base
```

Comment:
Can you use node:alpine instead of node node:18 to reduce the sieze of the image?

Reply:
No it won't bring any size difference, "node:alpine" will use latest node image which is 21 and by mentioning "18-alpine" it will use 18 version.
We need to test app with latest version then we can use `node:alpine`